examples/core/dof/app.js

```diff
@@ -4,6 +4,7 @@ import {
   Program, Texture2D, VertexArray, Buffer, isWebGL2
 } from 'luma.gl';
 import {Matrix4, radians} from 'math.gl';
+import {StatsWidget} from '@probe.gl/stats-widget'
```

Comment:
Longer term idea: We could make it so that the website injects this into all the examples. It would only be available when run as part of website, but we wouldn't need to add this code in 30 different files...